src/main/java/de/retest/recheck/ignore/RecheckIgnoreUtil.java

```diff
@@ -36,20 +37,24 @@ public static GlobalIgnoreApplier loadRecheckIgnore() {
 		try {
 			final LoadFilterWorker loadFilterWorker = new LoadFilterWorker( NopCounter.getInstance() );
 			return loadFilterWorker.load();
+		} catch ( final FileNotFoundException e ) {
+			logger.error( "Could not find recheck ignore file." );
 		} catch ( final Exception e ) {
-			logger.error( "Could not load recheck ignore file.", e );
-			return GlobalIgnoreApplier.create( NopCounter.getInstance() );
+			logger.error( "Exception loading recheck ignore file.", e );
 		}
+		return GlobalIgnoreApplier.create( NopCounter.getInstance() );
 	}
 
 	public static GlobalIgnoreApplier loadRecheckSuiteIgnore( final File ignoreFilesBasePath ) {
 		try {
 			final LoadFilterWorker loadFilterWorker =
 					new LoadFilterWorker( NopCounter.getInstance(), ignoreFilesBasePath );
 			return loadFilterWorker.load();
+		} catch ( final FileNotFoundException e ) {
+			logger.debug( "Ignoring missing suite ignore file." );
```

Comment:
We should also log the actual exception (like below):

```suggestion
			logger.debug( "Ignoring missing suite ignore file.", e );
```

Reply:
Same here.